en/console-and-shells.rst

```diff
@@ -1210,6 +1205,41 @@ look like:
         </arguments>
     </shell>
 
+Renaming Commands
+=================
+
+By default CakePHP will automatically discover all the commands in your
+application and its plugins. You may want to reduce the number of exposed
+commands, when building standalone console applications. You can use your
+Application's ``console()`` hook to limit which commands are exposed and rename
+the commands that are exposed::
+
+    namespace App;
+
+    use App\Shell\UserShell;
+    use App\Shell\VersionShell;
+    use Cake\Http\BaseApplication;
+
+    class Application extends BaseApplication
+    {
+        public function console($commands)
+        {
+            // Add by classname
```

Comment:
I find classnames simpler. But DI fans might like to build console commands from a container which lends itself to instances more.